src/python/pants/notes/2.13.x.md

```diff
@@ -1,5 +1,11 @@
 # 2.13.x Release Series
 
+## 2.13.0 (Sep 7, 2022)
+
+The first stable release of 2.13.0, with no changes since 2.13.0rc3.
+
+See our [blog post](https://blog.pantsbuild.org/introducing-pants-2-13/) for a summary of what's new in Pants 2.13.
```

Comment:
@chrisjrn we'll need to make sure your blog post uses this URL

Reply:
I've made the change in Ghost.